PRINCIPLES.md

```diff
@@ -10,20 +10,20 @@ These principles were derived from modern software operations but are rooted in
 
 ## Principles
 
-1. **The principle of declarative desired state**
+1. **Use declarative infrastructure**
```

Comment:
Since "infrastructure" can be seen in the context of physical infrastructure. I would explore different word for this.

"Use declarative pattern" or or "approach" or "model"

Reply:
Maybe it's just "Use Declarative State" or "Define State Declaratively"

Reply:
Updated